ecllibrary/std/File.ecl

```diff
@@ -959,4 +963,30 @@ EXPORT PromoteSuperFileList(set of varstring superNames, varstring addHead='', b
 EXPORT varstring GetEspURL(const varstring username = '', const varstring userPW = '') :=
     lib_fileservices.FileServices.GetEspURL(username, userPW);
 
+
+ /**
+ * Returns the path to the default Drop Zone
+ *
+ *
+ * @return              A string containing the path to the default DropZone.
```

Comment:
trivial: everywhere else it comments it says "Drop Zone"..

Reply:
Changed